src/encoded/schemas/changelogs/experiment.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for experiment.json
 
+### Schema version 21
+
+* Added *experiment_classification* property with enum "functional genomics assay", "functional characterization assay"
```

Comment:
How about "Added **required** property *experiment_classification* with enums "functional genomics assay" and "functional characterization assay"